packages/sync-actions/src/quote-requests.js

```diff
@@ -29,7 +26,7 @@ function createQuoteRequestsMapActions(
   return function doMapActions(
     diff: Object,
     newObj: Object,
-    oldObj: Object,
+    oldObj: Object
```

Comment:
Why did we remove the `,` here? Is is some auto-formatting regression?

Reply:
I think the "put a comma" rule only applies to objects and not function params